apis/python/tests/test_basic_spatialdata_io.py

```diff
@@ -0,0 +1,224 @@
+from urllib.parse import urljoin
+
+import numpy as np
+import pandas as pd
+import pyarrow as pa
+import pytest
+
+import tiledbsoma as soma
+from tiledbsoma import _factory
+
+spatial_outgest = pytest.importorskip("tiledbsoma.io.spatial.outgest")
```

Comment:
Do we still need to support skipping here?

Reply:
Yes. The spatial outgest uses optional packages that can cause an import error if the user didn't install them.